app/presenters/account_show_presenter.rb

```diff
@@ -1,16 +1,17 @@
 class AccountShowPresenter
   attr_reader :decorated_user, :decrypted_pii, :personal_key, :locked_for_session, :pii,
-              :sp_session_request_url, :sp_name
+              :sp_session_request_url, :sp_name, :account_reset_requested
```

Comment:
I figured that since the accessor was already a boolean that we could call it directly, this makes sense to keep up with convention throughout the app